GliaWidgets/SecureConversations/Transcript/TranscriptCoordinator.swift

```diff
@@ -15,7 +12,21 @@ extension SecureConversations {
         }
 
         func start() -> ChatViewController {
-            let model = TranscriptModel()
+            let model = TranscriptModel(
```

Comment:
Ah okay now I see that we are using the `ChatViewController` which probably have some expectations.

Reply:
The plan is at some point to swap it to `ChatViewModel`.